test/test.js

```diff
@@ -3,23 +3,28 @@ import test from 'ava';
 import 'babel-register';
 import 'babel-polyfill';
 
-import AnkiExport, { SEPARATOR } from '../src/index';
+import AnkiExport, {SEPARATOR} from '../src/index';
```

Comment:
Why you removed white spaces here and added them in `test/_helpers.js` https://github.com/ewnd9/anki-apkg-export/pull/8/files#diff-c193db11b3837e38bd24c06b98cb93ffR6 and at the bottom of this file https://github.com/ewnd9/anki-apkg-export/pull/8/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R68 ?

(white spaces are cool)